docs/bugs_by_version.json

```diff
@@ -502,4 +508,64 @@
         ], 
         "released": "2017-01-31"
     }
+}       "released": "2016-10-31"
+    }, 
+    "0.4.5": {
+        "bugs": [
+            "OneOfTwoConstructorsSkipped", 
```

Comment:
This seems to be broken.

Reply:
hm, something is wrong with the script...